config/300-eventing.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 The Knative Authors
+# Copyright 2021 The Knative Authors
```

Comment:
I don't think we bump these usually, do we?

Reply:
Not sure - some did before? I am fine in reverting

Reply:
no need to change the year. The year is only added when the file is created.